source/_integrations/ohme.markdown

```diff
@@ -143,6 +143,47 @@ The `ohme.set_price_cap` action is used to set the price cap threshold. This can
 | `config_entry`         | No       | The config entry of the account to apply the price cap to.  |
 | `price_cap`            | No       | Threshold in 1/100ths of your local currency.               |
 
+
+## Use cases
+
+This integration can be used to show charger status on your dashboards. It can also be used as part of automations for solar and battery storage.
+
+
+## Examples
+
+### Send a notification when a vehicle is plugged in:
+{% raw %}
+```yaml
+# Example automation
+mode: single
+triggers:
+  - trigger: state
+    entity_id:
+      - sensor.ohme_home_pro_status
+    from: unplugged
+conditions: []
+actions:
+  - action: notify.mobile_app_iphone
+    data:
+      message: Vehicle plugged in
+
+```
+{% endraw %}
+
+
+## Troubleshooting
+
+### The device is unavailable
+You may need to power cycle your charger. Please see the [manufactuer's guidance](https://ohme-ev.com/support/my-charger-is-offline/) for the procedure.
+
```

Comment:
Please see our integration quality scale documentation on how this section should be formatted:

https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-troubleshooting#example-implementation